src/shared/src/components/molecules/AltinnMobileTableItem.tsx

```diff
@@ -122,11 +125,11 @@ const useStyles = makeStyles({
       },
     },
     '&:hover': {
-      background: theme.altinnPalette.primary.red,
+      background: '#D5203B',
       color: theme.altinnPalette.primary.white,
     },
     '&:focus': {
-      outlineColor: theme.altinnPalette.primary.red,
+      outlineColor: '#D5203B',
```

Comment:
It seems like the only other usage of the color primary.red is borders of some inputs like the date-picker component for validation errors. `#D5203B` seems to be the color used for validation-error borders on the input in the altinn-design-system so it would probably be a good idea to update for all red colours here so that the input-fields have the same red-color when invalid.

Reply:
Agreed. Maybe just update the value in the theme then? 🤔